apps/autorelayer-interop/src/index.ts

```diff
@@ -1,3 +1,2 @@
-import { RelayerApp } from '@/app.js'
```

Comment:
lets leave this within `index.ts` just so that all the "start" commands are consistent `tsc src/index.ts`

Within the `package.json` we can just export the `app.ts` file

Reply:
fixed